app/models/category.rb

```diff
@@ -93,6 +92,8 @@ class Category < ActiveRecord::Base
   after_commit :trigger_category_updated_event, on: :update
   after_commit :trigger_category_destroyed_event, on: :destroy
 
+  after_save_commit :index_search
```

Comment:
In all other instances we use this form:

```suggestion
  after_commit :index_search, on: :save
```

Reply:
Not exactly....

We use the form `after_commit :index_search, on: [:create, :update].

`on: :save` is not valid?

We also probably use `after_commit` for because `after_save_commit` is a new to rails 6 it wasn't really an option to use before in most of our code.

Reply:
Oh, you're right, there's no `:save` option… 😬 So it's a choice between consistency and readability/compactness. 🤔 I'm fine with either one then 😛